packages/truffle-core/lib/commands/migrate.js

```diff
@@ -28,7 +32,7 @@ var command = {
   },
   help: {
     usage:
-      "truffle migrate [--reset] [-f <number>] [--network <name>] [--compile-all] [--verbose-rpc] [--interactive]",
+      "truffle migrate [--reset] [--f <number>] [--to <number>] [--network <name>] [--compile-all] [--verbose-rpc] [--interactive]",
```

Comment:
Is it really `--f` and not `--from`?

Reply:
Could we make it both `-f` and `--from`?

Reply:
Wow, `--from` works, but I can't find it specified anywhere in the codebase.

Reply:
Oh! Found it. Hidden in `truffle-migrate/index.js` https://github.com/trufflesuite/truffle/blob/develop/packages/truffle-migrate/index.js#L63

Reply:
although... I don't think that line does what it's intended to do 😂

Reply:
Okay, nm. Forget what I said above, `--from` doesn't work as a cmd line `truffle migrate` option.